frontend/src/components/common/SearchAndFilters.tsx

```diff
@@ -65,6 +64,7 @@ const SearchAndFilters = ({ setLogRecords }: Props): React.ReactElement => {
         dateRange,
         tags ? [tags] : [],
         flagged,
+        false, // do not return all data
```

Comment:
Relates back to my comment above, there would be no need to do this if we passed in an object